### ISSUE TITLE
Some of the line heights and spacing between text is too small.

### ISSUE THREAD

**Issue by sssoz** [oaworks/oa-works#124]
This fails [WCAG 2.1 success criterion 1.4.12: **text spacing**](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html).

**Comment by thinkbulecount2** [comment]
I suggest calculating the line-height as a relative measurement (i.e. either as em or percentage) depending on the font-size, if that makes sense. Based on WCAG 2.1's criterion:
```Line height (line spacing) to at least 1.5 times the font size;
Spacing following paragraphs to at least 2 times the font size;
Letter spacing (tracking) to at least 0.12 times the font size;
Word spacing to at least 0.16 times the font size.

**Comment by sssoz** [comment]
@thinkbulecount2 Agreed. But I’m also a bit worried about how it would look aesthetically. This is something we could pair up and do together as it concerns both design choices and a11y.

**Comment by sssoz** [comment]
See: https://github.com/oaworks/oa-works/blob/main/src/_includes/css/typography.css

And potentially add variables here: https://github.com/oaworks/oa-works/blob/main/src/_includes/css/variables.css

**Comment by thinkbulecount2** [comment]
What should we do with this issue, if we're moving to Tailwind in the future? @sssoz

**Comment by sssoz** [comment]
There are no specific plans to switch OA.Works to TailwindCSS — the main site isn’t a priority. We’re just using it for OAreport and potentially OAsupport for now.
